python/cucim/src/cucim/core/operations/expose/tests/test_expose.py

```diff
@@ -0,0 +1,21 @@
+import cucim.core.operations.expose.transform as expose
+
+color_jitter = expose.color_jitter
+image_flip = expose.image_flip
+image_rotate_90 = expose.image_rotate_90
+scale_intensity = expose.scale_intensity_range
+zoom = expose.zoom
+rand_zoom = expose.rand_zoom
+rand_image_flip = expose.rand_image_flip
+rand_image_rotate_90 = expose.rand_image_rotate_90
```

Comment:
Would suggest just `import`ing them as expected

```suggestion
from cucim.core.operations.expose.transform.expose import (
    color_jitter,
    image_flip,
    image_rotate_90,
    scale_intensity,
    zoom,
    rand_zoom,
    rand_image_flip,
    rand_image_rotate_90,
)
```